st2tests/st2tests/config.py

```diff
@@ -139,12 +139,18 @@ def _override_scheduler_opts():
 
 def _override_coordinator_opts(noop=False):
     driver = None if noop else "zake://"
+
+    redis_host = os.environ.get("ST2TESTS_REDIS_HOST", False)
+    if redis_host:
+        redis_port = os.environ.get("ST2TESTS_REDIS_PORT", "6379")
+        driver = f"redis://{redis_host}:{redis_port}"
```

Comment:
`ST2TESTS_*` is the convention I've been using for vars that configure tests:
https://github.com/StackStorm/st2/blob/78c3248bf8c887add3b5bf569983f2c6f4421c6e/st2tests/st2tests/config.py#L89-L90
https://github.com/StackStorm/st2/blob/78c3248bf8c887add3b5bf569983f2c6f4421c6e/st2tests/st2tests/config.py#L107-L109